doc/howto/images_manage.md

```diff
@@ -1,20 +1,105 @@
+
+    lxc image list images: debian
+
+You can specify several filters as well.
+For example, to show all 64-bit Debian images, enter the following command:
+
+    lxc image list images: debian amd64
+
+To filter for properties other than alias or fingerprint, specify the filter in `<key>=<value>` format.
+For example:
+
+    lxc image list images: debian architecture=x86_64
+
+## View image information
+
+To view information about an image, enter the following command:
+
+    lxc image info <image_ID>
+
+As the image ID, you can specify either the image's alias or its fingerprint.
+For a remote image, remember to include the remote server (for example, `images:ubuntu/22.04`).
+
+To display only the image properties, enter the following command:
+
+    lxc image show <image_ID>
+
+You can also display a specific image property (located under the `properties` key) with the following command:
+
+    lxc image get-property <image_ID> <key>
+
+For example, to show the release name of the official Ubuntu 22.04 image, enter the following command:
+
+    lxc image get-property ubuntu:22.04 release
+
+(images-manage-edit)=
+## Edit image properties
+
+To set a specific image property that is located under the `properties` key, enter the following command:
+
+    lxc image set-property <image_ID> <key>
+
+To edit the full image properties, including the top-level properties, enter the following command:
+
+    lxc image edit <image_ID>
+
+## Configure image aliases
+
+Configuring an alias for an image can be useful to make it easier to refer to an image, since remembering an alias is usually easier than remembering a fingerprint.
+Most importantly, however, you can change an alias to point to a different image, which allows creating an alias that always provides a current image (for example, the latest version of a release).
+
+You can see some of the existing aliases in the image list.
+To see the full list, enter the following command:
+
+    lxc image alias list
+
+You can directly assign an alias to an image when you {ref}`copy or import <images-copy>` or {ref}`publish <images-create-publish>` it.
+Alternatively, enter the following command:
+
+    lxc image alias create <alias> <image_fingerprint>
+
+You can also delete an alias:
+
+    lxc image alias delete <alias>
+
+To rename an alias, enter the following command:
+
+    lxc image alias rename <alias> <new_alias>
+
+If you want to keep the alias name, but point the alias to a different image (for example, a newer version), you must delete the existing alias and then create a new one.
+
+(images-manage-export)=
+## Export an image to a file
+
+Images are located in the image store of your local server or a remote LXD server.
+You can export them to a file though.
+This method can be useful to back up image files or to transfer them to an air-gapped environment.
+
+To export a container image to a file, enter the following command:
+
+    lxc image export [<remote>:]<image> [<output_location>]
```

Comment:
Maybe `<output_directory_path>` here? This would help to indicate that it requires a directory path (as opposed to an image file name, which one may expect when exporting an image).